nixos/modules/system/boot/binfmt.nix

```
@@ -248,6 +248,8 @@ in {
         description = ''
           List of systems to emulate. Will also configure Nix to
           support your new systems.
+          Be careful as a build can execute all emulated systems at the same time.
```

Maybe this is better:
```suggestion
          Warning: the builder can execute all emulated systems within the same build, which introduces impurities in the case of cross compilation.
```